### ISSUE TITLE
Build failing for 1.7.0-RC-03-snapshot from source

### ISSUE THREAD

**Issue by jholland-usgs** [iris-edu/stationxml-validator#106]
Attempting to build the latest version results in the below error:
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project stationxml-validator: 
Could not resolve dependencies for project edu.iris.dmc:stationxml-validator:jar:1.7.0-RC-03-SNAPSHOT: 
Could not find artifact edu.iris.dmc:stationxml-seed-converter:jar:2.1.0-RC-03-SNAPSHOT in ossrh (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
[ERROR] 
```

Thank you for looking into this.

**Comment by timronan** [comment]
edu.iris.dmc:stationxml-seed-converter:jar:2.1.0-RC-03-SNAPSHOT hadn't been deployed. Please try the build again. 

The new build should resolved issue #107. Let me know if there are anymore issues.

**Comment by jholland-usgs** [comment]
Build works now, thank you.
